From 5369c963106f2c8d53d36787b51e4d19f8057902 Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Thu, 14 Mar 2019 13:07:37 +0000
Subject: [PATCH 13/13] Add Fedora 30 support

In this release, if specified, we default to UEFI support.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 oz/Fedora.py          |  5 +++++
 oz/auto/Fedora30.auto | 23 +++++++++++++++++++++++
 2 files changed, 28 insertions(+)
 create mode 100644 oz/auto/Fedora30.auto

diff --git a/oz/Fedora.py b/oz/Fedora.py
index 118fd84..34c4fe7 100644
--- a/oz/Fedora.py
+++ b/oz/Fedora.py
@@ -117,6 +117,11 @@ class FedoraConfiguration(object):
 
 
 version_to_config = {
+    '30': FedoraConfiguration(has_virtio_channel=True, use_yum=False,
+                              use_dev_cdrom_device=True, createpart=False,
+                              directkernel="cpio", default_netdev='virtio',
+                              default_diskbus='virtio', brokenisomethod=False,
+                              haverepo=True, useuefi=True),
     '29': FedoraConfiguration(has_virtio_channel=True, use_yum=False,
                               use_dev_cdrom_device=True, createpart=False,
                               directkernel="cpio", default_netdev='virtio',
diff --git a/oz/auto/Fedora30.auto b/oz/auto/Fedora30.auto
new file mode 100644
index 0000000..1dae86a
--- /dev/null
+++ b/oz/auto/Fedora30.auto
@@ -0,0 +1,23 @@
+install
+text
+keyboard us
+lang en_US.UTF-8
+skipx
+network --device ens3 --bootproto dhcp
+rootpw %ROOTPW%
+firewall --disabled
+authconfig --enableshadow --enablemd5
+selinux --enforcing
+timezone --utc America/New_York
+bootloader --location=mbr --append="console=tty0 console=ttyS0,115200"
+
+zerombr
+clearpart --all --initlabel
+autopart --type=lvm
+
+reboot
+
+%packages
+@core
+
+%end
-- 
2.20.1

